src/actions/getProjectsArchive.js

```diff
@@ -26,9 +26,15 @@ export default async function getProjectsArchive() {
         bucket: AWS_ARTISAN_BUCKET,
         key: `${ARCHIVE_PROJECTS_DIRECTORY}/${d}/`,
       });
+      const illosList = await s3.list({
+        bucket: AWS_ARTISAN_BUCKET,
+        prefix: `${ARCHIVE_PROJECTS_DIRECTORY}/${d}/`,
+      });
+      const illos = await fetchIlloMeta(illosList);
```

Comment:
Feels like `fetchProjectsArchive` should also just come with illos no? Is there a particular case where we'd want the project without all the illos?

Reply:
Me neither. Bundled it up into `fetchProjectArchive` with a bit of refactoring.